nemo_rl/models/generation/vllm/vllm_backend.py

```
@@ -125,8 +125,51 @@ def _maybe_process_fp8_kv_cache(self) -> None:
+        """Apply weight name mapping if LoRA is enabled."""
+        new_weights = []
+        for name, w in weights:
+            new_name = self.map_param_name(name)
+            new_weights.append((new_name, w))
+        return new_weights
+
     @wrap_with_nvtx_name("vllm_internal_worker_extension/update_weights_via_ipc_zmq")
-    def update_weights_via_ipc_zmq(self) -> bool:
+    def update_weights_via_ipc_zmq(
+        self,
+        lora_config: dict[str, Any] = {},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)